modules/client/examples/05-encoders-decoders/01-grant-permission.ts

```diff
@@ -0,0 +1,69 @@
+/* MARKDOWN
```

Comment:
There's already one file with index `1` right above

Reply:
Fixed in 2f284e5bbe93a665e8ffd13477ee3fc44d81b3be